connectors/sources/box.py

```diff
@@ -391,9 +390,9 @@ def _pre_checks_for_get_content(
             )
             return False
 
-        if attachment_size > FILE_SIZE_LIMIT:
+        if attachment_size > self.framework_config.max_file_size:
             self._logger.warning(
-                f"File size {attachment_size} of file {attachment_name} is larger than {FILE_SIZE_LIMIT} bytes. Discarding file content"
+                f"File size {attachment_size} of file {attachment_name} is larger than {self.framework_config.max_file_size} bytes. Discarding file content"
```

Comment:
I was also wondering if it's possible to move this condition elsewhere, so that each connector would not call it?

Best I could think of was that rather than doing comparison manually, make the class "richer" and do something like:

```
if self.framework_config.file_size_supported(attachment_size):
    # download
```

How do you feel?

Reply:
I'm not going to tackle that as part of this - my focus for this PR is making the existing logic configurable. I believe there was debate previously about moving this kind of thing to the Extraction utils, but that ended up being rejected, and I don't want to re-open that discussion at the moment.